packages/validator/docs/data/actions.ts

```diff
@@ -0,0 +1,270 @@
+export const copyTemplate =
```

Comment:
There is 25 tests, each test has a template, valid specs, invalid specs and expected output for invalid specs, which sums up to 100 files for all the tests. It might be practical to just run test with one command, but i think it would create way too many files.
Second reason is currently used library does not support snippet injections from json files.